### PR TITLE
Remove specific number of steps in update docs

### PR DESCRIPTION
**Pull request by Vexed01**
### Description of the changes

Change "these 4" to "a few" in the Updating Red docs because 4 is a lie as there are 5.

### CODE REVIEW

File: docs/update_red.rst [hunk 1]
```diff
@@ -31,7 +31,7 @@ Red 3.2.0 or newer
 Windows
 -------
 
-If you have Red 3.2.0 or newer, you can upgrade by following these 4 easy steps:
+If you have Red 3.2.0 or newer, you can upgrade by following a few easy steps:
```
Comment (Jackenmen) [inline docs/update_red.rst]:
```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these steps:
```

File: docs/update_red.rst [hunk 2]
```diff
@@ -59,7 +59,7 @@ If you have Red 3.2.0 or newer, you can upgrade by following these 4 easy steps:
 Linux & Mac
 -----------
 
-If you have Red 3.2.0 or newer, you can upgrade by following these 4 easy steps:
+If you have Red 3.2.0 or newer, you can upgrade by following a few easy steps:
```
Comment (Jackenmen) [inline docs/update_red.rst]:
```suggestion
If you have Red 3.2.0 or newer, you can upgrade by following these steps:
```

**Review by Jackenmen — COMMENTED** [review]
What do you think about this?

**Comment by Vexed01** [comment]
Certainly makes it simpler

**Review by Jackenmen — APPROVED** [review]

**Pull request merged**